lib/liquid/autoescape/liquid_ext/block_body.rb

```diff
@@ -0,0 +1,47 @@
+require "liquid"
+require "liquid/autoescape"
+require "liquid/autoescape/template_variable"
+require "liquid/autoescape/safe_string"
+
+module Liquid
+  class BlockBody
+
+    def render_node(context, output, node)
+      if !node.is_a? Variable
+        output << BlockBody.render_node(context, +"", node)
```

Comment:
I think this kinda defeats the whole "single output buffer" idea, why not this?

```suggestion
        BlockBody.render_node(context, output, node)
```

Reply:
Makes sense